pkg/tsdb/azuremonitor/azuremonitor.go

```diff
@@ -54,7 +54,30 @@ func ProvideService(httpClientProvider *httpclient.Provider) *Service {
 	return s
 }
 
+func handleDeprecatedQueryTypes(req *backend.QueryDataRequest) *backend.QueryDataResponse {
+	// Logic to handle deprecated query types that haven't been migrated
+	responses := backend.Responses{}
+	for _, q := range req.Queries {
+		if q.QueryType == "Application Insights" || q.QueryType == "Insights Analytics" {
+			responses[q.RefID] = backend.DataResponse{
+				Error:       fmt.Errorf("query type: '%s' is no longer supported. Please migrate this query", q.QueryType),
```

Comment:
Should we give them more info on migrating the query? Are the ways to migrate too varied?

Reply:
I've added a link to the docs detailing the migration info 😊